### DISCUSSION TITLE
Sprint 3 meeting | 2022-10-18

### DISCUSSION THREAD

**Discussion by HuangJunye** [QPong/planning#32] [Meetings]
## Agenda

- [x] Sprint review (10 min)
- [x] Sprint retro (10 min)
- [x] Sprint planning (10 min)
- [x] Purchasing PICO-8
- [x] Q&A (5 min)
- [x] Game dev resources and news (5 min)
- [x] Vacations or other commitments
  - [x] Junye is going on vacations (Oct 22-Nov 1)
  - [x] Siddharth is on vacations (Oct 21-Oct 28)
  - [x] Reschedule next Sprint meeting => Nov 3 2pm CET
  - [x] Future sprint meetings 2pm CET due to winter time change.

**Comment by HuangJunye** [comment]
## Review

[Last sprint planning](https://github.com/QPong/planning/discussions/13#discussioncomment-3794908)

### Team
- #33 

### Dayeong

### Unathi

### Siddharth

### Junye 

### Follow up
- [ ] #3 
- [ ] #16 
- [ ] #14
- [x] #20
- [x] #21

**Reply by tula3and** [reply]
#22

**Reply by Unathi-Skosana** [reply]
See [comment](https://github.com/QPong/planning/issues/16#issuecomment-1282328437)

**Comment by HuangJunye** [comment]
## Retro

### What went well


### What didn't go well


### What to improve?

**Reply by tula3and** [reply]
### What went well

- Complete this sprint's task
- Had a hackathon

### What didn't go well

- Still not clear to connect the server and pico-8

### What to improve?

- Make a simple MVP for testing server communication

**Reply by Unathi-Skosana** [reply]
## What went well

- Hackathon was a success.

## What didn't go well

- The workflow of locally editing pico8 source code, and the once-exported compiled JS source code to reflect changes in the pico8 source is challenging.

## What to improve?

- I think I will have to acquiesce and just purchase a pico8.

**Reply by HuangJunye** [reply]
## What went well

- Hackathon was great. I love hacking together

## What didn't go well

- The progress is a bit slow. Blocked by PICO-8 purchase and server code

## What to improve?
- Purchase PICO-8 license
- Plan another hackathon sometime? Nov 7 1:30pm-4:30pm CET.

**Comment by HuangJunye** [comment]
## Planning

### Dayeong

### Unathi

### Siddharth

### Junye

**Reply by tula3and** [reply]
- Make a simple MVP for testing server communication
- Modify qpong.p8 to make 2-player mode

**Reply by HuangJunye** [reply]
Siddharth to implement noise bomb in qpong.p8
